website/project/decorators.py

```diff
@@ -329,3 +330,15 @@ def wrapped(*args, **kwargs):
 
     # Return decorator
     return wrapper
+
+
+def dev_only(func):
```

Comment:
But why should the Oauth Applications UI views be dev-only? `dev_only` seems to be for views that are for developers of the OSF. Oauth Applications are for client applications.